app/ui/browser-modern/views/browser/page/index.jsx

```diff
@@ -204,13 +204,30 @@ class Page extends Component {
 
     this.webview.addEventListener('ipc-message', e => {
```

Comment:
Mousedown would be sufficient for this, I think. Can propogate as 'click' if needed